axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryBus.java

```diff
@@ -608,6 +644,11 @@ public Builder localSegment(QueryBus localSegment) {
             return this;
         }
 
+        public Builder localSegmentShortCut(boolean localSegmentShortCut) {
```

Comment:
Ow, and we really do not some JavaDoc on this method to clarify the user what it does!